benchmark/plan.sh

```diff
@@ -19,37 +19,52 @@ pkg_build_deps=(
   core/gcc
   core/git
   core/googletest
-  core/googlemock
 )
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
 
-BUILDDIR="build"
+do_begin() {
+  export HAB_ENV_CMAKE_FIND_ROOT_PATH_SEPARATOR=";"
+  export HAB_ENV_CMAKE_FIND_ROOT_PATH_TYPE="aggregate"
+}
+
+do_setup_environment() {
+  set_buildtime_env BUILD_DIR "_build"
+
+  # this allows cmake users to utilize `CMAKE_FIND_ROOT_PATH` to find various cmake configs
+  push_runtime_env CMAKE_FIND_ROOT_PATH "${pkg_prefix}/lib/cmake/benchmark"
+}
 
 do_prepare() {
-  mkdir -p "$BUILDDIR"
+  mkdir -p "${BUILD_DIR}"
```

Comment:
In some places you're using `"${BUILD_DIR}"` and in other places without the braces `"$BUILD_DIR"`

I'd love to see it done one way. Personally I prefer with braces `"${}BUILD_DIR}"` as its "safer".

Reply:
agree and resolved to be the `"${BUILD_DIR}"`